src/components/backend-ai-usersettings-general-list.ts

```diff
@@ -116,14 +125,16 @@ export default class BackendAiUsersettingsGeneralList extends BackendAIPage {
           width: 75px;
         }
 
-        .setting-item wl-button {
-          --button-bg: transparent;
-          --button-bg-hover: var(--paper-teal-100);
-          --button-bg-active: var(--paper-teal-100);
-          --button-bg-disabled: #cccccc;
-          --button-color: var(--paper-teal-100);
-          --button-color-hover: var(--paper-teal-100);
-          --button-color-disabled: #cccccc;
+
+        .ssh-keypair {
+          margin-right:10px;
+          width: 450px;
+          min-height:100px;
+          overflow-y:scroll;
+          white-space:pre-wrap;
+          word-wrap:break-word;
+          font-size:10px;
+          scrollbar-width: none; /* firefox */
```

Comment:
This is also a minor issue, but please be consistent with the space after colon(`:`). This repository prefers one space after `:`, except for inline-styles.